.travis.yml

```diff
@@ -1,16 +1,28 @@
 os: linux
 language: rust
 cache: cargo
+before_cache:
+  - find target/debug -type f -maxdepth 1 -delete
+  - rm -rf target/.rustc_info.json
+  - rm -rf target/debug/examples
+  - rm -rf target/debug/incremental
```

Comment:
Why do you want to remove incremental? I thought the whole point of caching is making use of incremental compilation.
Also, I think examples should be preserved too.

Reply:
The issue is the clap source code will keep changing and the cache will keep getting bigger and bigger and soon we have timeouts in Travis CI. So, we are limiting cache only to the compiled deps.

Reply:
Neverminf